### DISCUSSION TITLE
Help Needed: Bottomsheet Modals in a Stack Screen with 'formSheet' Presentation Style

### DISCUSSION THREAD

**Discussion by Karthik-B-06** [gorhom/react-native-bottom-sheet#1718] [General]
I am trying to use Bottomsheet Modal inside a Stack Screen, which has the presentation style set to `formSheet`. The Bottomsheet is not appearing on top of this screen but is rendered behind it. How can I get the Bottomsheet Modal to work on this screen?

**Comment by bardonadam** [comment]
Facing the same issue
